build.sbt

```diff
@@ -119,11 +119,12 @@ unmanagedResourceDirectories in Compile += baseDirectory.value / "src" / "main"
 (scalastyleConfig in Test) := baseDirectory.value / "scalastyle-test-config.xml"
 scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
 
+// Packaging
 mainClass in Compile := Some("io.iohk.ethereum.App")
-
+discoveredMainClasses in Compile := Seq("io.iohk.ethereum.mallet.main.Mallet")
```

Comment:
This doesn't seem to work on my machine:

```
 % ./bin/mallet
Error: Could not find or load main class io.iohk.ethereum.mallet.main.Mallet
```

Reply:
Yup, once I realaized that this probably needs to be adjusted, @mirkoAlic confirmed for me that mallet can be removed so there will be a separate PR soon I think